src/handlers.ts

```diff
@@ -0,0 +1,11 @@
+
+export class ProsopoEnvError extends Error {
```

Comment:
Have assigned https://github.com/prosopo-io/provider/issues/21 to you based on this as I think it covers it.